### ISSUE TITLE
Javadoc hosting on GitHub pages?

### ISSUE THREAD

**Issue by JakeWharton** [google/TestParameterInjector#1]
You can have a GitHub Action build the docs and automatically deploy to a `site` branch (or any branch name) which is enabled for GitHub Pages.

We use https://github.com/JamesIves/github-pages-deploy-action for this on a few projects.

**Comment by nymanjens** [comment]
Thanks for the information! Currently, we only have the one README.md file, which IMO seems enough for now. Is there something concretely you are missing that a site branch would give you?

**Comment by JakeWharton** [comment]
The README says something along the lines of "See the Javadoc of @TestParameter..." about the supported types. When I shared the tool I linked to the README section but someone had asked about the canonical set of supported types and I had to link the source of the documentation rather than the rendered documentation.

I agree it's not absolutely necessary, but with GitHub Actions it's quite trivial to set up. I can even send a PR if you want!
